.github/workflows/publish_images.yaml

```diff
@@ -0,0 +1,120 @@
+name: Publish images
+
+on:
+  workflow_dispatch:
+
+  push:
+    branches:
+      - main
+      - brent/base
+    paths-ignore:
+      - "*.md"
+
+  schedule:
+    - cron: '0 7 * * 2'
```

Comment:
For confirmation: we'll be publishing an image every Tuesday at 07:00 UTC, that's what we want?

Reply:
The time picked was arbitrary, and it stepped on me during testing a couple times so I bounced it around :)

The idea is to automatically pick up updated .debs to automatically include security fixes. I'd like it scheduled, but I don't care at all about when.

Reply:
👍